toc/working-groups/paketo.md

```diff
@@ -0,0 +1,40 @@
+# Paketo: Working Group Charter
+
+## Mission
+
+Provides Paketo buildpacks for servers, languages, and frameworks popular with application developers.
+
+
+## Goals
+
+- Provides end users with stable, reliable and performant buildpacks with which to build their application.
+- Provides end users with buildpacks that are compatible with multiple platforms (such as Cloud Foundry, Docker, and Kubernetes).
+- Provides documentation to enable platforms to adopt Paketo buildpacks and maintain compatibility with Paketo buildpacks.
+
+## Scope
+
+- Provide the community with regular releases of Paketo buildpacks.
+- Provide timely releases to address CVEs.
+- Maintain public roadmaps for the development of Paketo.
+- Provide the community with technical and end user documentation.
+- Collaborate with App Runtime Interfaces WG to enable Paketo buildpacks to have first-class support in Cloud Foundry.
+
+## Non-Goals
+
+- Maintaining V2 (classic) buildpacks
+
+## Working Group Governance Structure
+
+[Working group governance structure](https://github.com/paketo-buildpacks/community/blob/main/GOVERNANCE.md)
```

Comment:
Should we check for potential convergence of this document with the [working groups](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/WORKING-GROUPS.md) and [roles](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md) documents?

If the answer is 'no', in favor of Paketo's independence as a separate project, should we at least check for potential incompatibilities?

At the very least we should make sure it is compatible (and probably mentions) the [principles](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md).

Reply:
I do not see an immediate conflict. 

I still see potential for alignment. E.g. the *steering commitee* members seems to be roughly what the *Paketo working-group leads* would be. Subteam *maintainers* seem to be *approvers* and *contributors* members.

Having said that, in order to let people just work, I'll approve this PR and we'll just figure out what the relationship the Cloud Foundry working groups and the Paketo working group will develop and what the role of the TOC will be in that relationship.